packages/inngest/src/api/api.ts

```diff
@@ -408,6 +409,80 @@ export class InngestApi {
       });
   }
 
+  async updateMetadata(
+    args: {
+      target: MetadataTarget;
+      metadata: Array<{
+        kind: string;
+        op: string;
+        values: Record<string, unknown>;
+      }>;
+    },
+    options?: {
+      headers?: Record<string, string>;
+    },
+  ): Promise<Result<void, ErrorResponse>> {
+    const url = await this.getTargetUrl(
```

Comment:
I added `this.req()` a little while ago that can help clean up some of this boilerplate

https://github.com/inngest/inngest-js/blob/94d80ab4ffbd4c72d35b0033e2afcee63b425f2b/packages/inngest/src/api/api.ts#L152-L178